### PR TITLE
Syntax highlighting support

### PR DESCRIPTION
**Pull request by simonbaird**
Note that we use the Antora method of client side syntax highlighting with the Hugo build to be consistent.

The antora build already had working syntax highlighting, but now it has a different color scheme.

I picked a theme I liked from https://highlightjs.org/static/demo/

### CODE REVIEW

File: website/content/posts/introducing-the-enterprise-contract.md [hunk 1]
```diff
@@ -23,16 +23,16 @@ ensures the same image is being verified across all steps preventing
 tools that can do this. I’ll use [skopeo](https://github.com/containers/skopeo) since I’m familiar
 with it:
 
-```text
+{{<code bash>}}
```
Comment (lcarva) [inline website/content/posts/introducing-the-enterprise-contract.md:26]: The rendered markdown appears to have this class `language-text`. Can we just use that instead?

Comment (zregvart) [inline website/content/posts/introducing-the-enterprise-contract.md:26]: Just to check ` ```bash ` doesn't work here?

File: common/highlightjs-theme.css [hunk 2]
```diff
@@ -0,0 +1 @@
...ype{color:#82aaff}.hljs-literal{color:#ff5874}.hljs-number{color:#f78c6c}.hljs-regexp{color:#5ca7e4}.hljs-string{color:#ecc48d}.hljs-subst{color:#d3423e}.hljs-symbol{color:#82aaff}.hljs-class{color:#ffcb8b}.hljs-function{color:#82aaff}.hljs-title{color:#dcdcaa;font-style:italic}.hljs-params{color:#7fdbca}.hljs-comment{color:#637777;font-style:italic}.hljs-doctag{color:#7fdbca}.hljs-meta,.hljs-meta .hljs-keyword{color:#82aaff}.hljs-meta .hljs-string{color:#ecc48d}.hljs-section{color:#82b1ff}.hljs-attr,.hljs-name,.hljs-tag{color:#7fdbca}.hljs-attribute{color:#80cbc4}.hljs-variable{color:#addb67}.hljs-bullet{color:#d9f5dd}.hljs-code{color:#80cbc4}.hljs-emphasis{color:#c792ea;font-style:italic}.hljs-strong{color:#addb67;font-weight:700}.hljs-formula{color:#c792ea}.hljs-link{color:#ff869a}.hljs-quote{color:#697098;font-style:italic}.hljs-selector-tag{color:#ff6363}.hljs-selector-id{color:#fad430}.hljs-selector-class{color:#addb67;font-style:italic}.hljs-selector-attr,.hljs-selector-pseudo{color:#c792ea;font-style:italic}.hljs-template-tag{color:#c792ea}.hljs-template-variable{color:#addb67}.hljs-addition{color:#addb67ff;font-style:italic}.hljs-deletion{color:#ef535090;font-style:italic}
```
Comment (zregvart) [inline common/highlightjs-theme.css:1]: Duplicate of `common/css/hjiglightjs-theme.css`?

Comment (simonbaird) [reply common/highlightjs-theme.css:1]: ah, thanks.

File: website/content/posts/introducing-the-enterprise-contract.md (hunk 1 above)
Comment (simonbaird) [reply website/content/posts/introducing-the-enterprise-contract.md:26]: Not without some customization of the highlightjs on load behavior. (Which is doable, but maybe for later.)

Comment (simonbaird) [reply website/content/posts/introducing-the-enterprise-contract.md:26]: Not without some customization of the highlightjs on load behavior. (Which is doable, but maybe for later.)

**Pull request merged**